content/en/docs/instrumentation/java/libraries.md

```diff
@@ -23,3 +23,180 @@ some instrumentation libraries for your dependencies standalone. To find out
 which standalone instrumentation libraries are available, take a look at
 [this list](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/supported-libraries.md#libraries--frameworks).
 Follow the instructions of each instrumentation library to set them up.
+
+## Example app preparation {#example-app}
+
+We will instrument an HTTP client application using library instrumentation
+which will make a call to HTTP server.
+
+You can use the dice example app as HTTP server from
+[Getting Started](/docs/instrumentation/java/getting-started/) or you can create
+your own HTTP server.
+
+### Dependencies {#example-app-dependencies}
+
+To begin, set up an environment in a new directory called
+`java-simple-http-client`. Within that directory, create a file called
+`build.gradle.kts` with the following content:
+
+```kotlin
+plugins {
+  id("java")
+  id("application")
+}
+
+application {
+  mainClass.set("otel.SampleHttpClient")
+}
+
+sourceSets {
+  main {
+    java.setSrcDirs(setOf("."))
+  }
+}
+
+repositories {
+  mavenCentral()
+}
+
+dependencies {
+    implementation("io.opentelemetry:opentelemetry-api:1.32.0");
+    implementation("io.opentelemetry:opentelemetry-sdk:1.32.0");
+    implementation("io.opentelemetry:opentelemetry-exporter-logging:1.32.0");
+    implementation("io.opentelemetry.semconv:opentelemetry-semconv:1.22.0-alpha");
+    implementation("io.opentelemetry:opentelemetry-sdk-extension-autoconfigure:1.32.0");
```

Comment:
Add `autoconfigure` to the cspell ignore list in the front matter (see other pages how this looks like, e.g. https://github.com/open-telemetry/opentelemetry.io/blob/main/content/en/docs/instrumentation/java/exporters.md?plain=1#L4)

Reply:
done